client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SnapshotClientDocumentationIT.java

```diff
@@ -367,6 +369,74 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testSnapshotSnapshotsStatus() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        createTestRepositories();
+        createTestSnapshots();
+
+        // tag::snapshots-status-request
+        SnapshotsStatusRequest request = new SnapshotsStatusRequest();
+        // end::snapshots-status-request
+
+        // tag::snapshots-status-request-repository
+        request.repository(repositoryName); // <1>
+        // end::snapshots-status-request-repository
+        // tag::snapshots-status-request-snapshots
+        String [] snapshots = new String[] {snapshotName};
+        request.snapshots(snapshots); // <1>
+        // end::snapshots-status-request-snapshots
+        // tag::snapshots-status-request-ignoreUnavailable
+        request.ignoreUnavailable(true); // <1>
+        // end::snapshots-status-request-ignoreUnavailable
+        // tag::snapshots-status-request-masterTimeout
+        request.masterNodeTimeout(TimeValue.timeValueMinutes(1)); // <1>
+        request.masterNodeTimeout("1m"); // <2>
+        // end::snapshots-status-request-masterTimeout
+
+        // tag::snapshots-status-execute
+        SnapshotsStatusResponse response = client.snapshot().snapshotsStatus(request, RequestOptions.DEFAULT);
+        // end::snapshots-status-execute
+
+        // tag::snapshots-status-response
+        List<SnapshotStatus> snapshotStatusesResponse = response.getSnapshots();
```

Comment:
could you go a bit deeper here and show users what they can retrieve from the response? We don't need to show everything but I would add what we'd expect users to generally do.